Source/IntegrationTests/TestFiles/LitTests/LitTest/git-issues/git-issue-1514.dfy

```diff
@@ -1,7 +1,6 @@
-// RUN: %testDafnyForEachCompiler "%s" --refresh-exit-code=0 -- --relax-definite-assignment
+// RUN: %testDafnyForEachCompiler "%s" --refresh-exit-code=0 -- --standard-libraries --relax-definite-assignment
```

Comment:
Thanks for cleaning these up. I think limited use of `--standard-libraries` in tests is fine as long as we don't end up adding it everywhere, and for new tests I'd prefer just adding the bare minimum necessary for the test (e.g. just `Option<T> = None | Some(value: T)` rather than the whole thing from the stdlibs)